src/components/chart/_macro.spec.js

```diff
@@ -193,6 +193,23 @@ describe('Macro: Chart', () => {
             });
         });
 
+        describe('GIVEN: Params: fallbackImageUrl', () => {
+            describe('WHEN: fallbackImageUrl is provided', () => {
+                const $ = cheerio.load(
+                    renderComponent('chart', {
+                        ...EXAMPLE_LINE_CHART_WITH_CONFIG_PARAMS,
+                        fallbackImageUrl: '/img/small/line-chart-screenshot.png',
+                        fallbackImageAlt: 'A description of the fallback image for screen readers',
+                    }),
+                );
+                const noScriptFallbackImage = $(`#fallback-image--chart-456`).html();
+                test('THEN: it renders the fallback image', () => {
+                    expect(noScriptFallbackImage).toContain('/img/small/line-chart-screenshot.png');
+                    expect(noScriptFallbackImage).toContain('alt="A description of the fallback image for screen readers"');
```

Comment:
I think this should be split out into another test as its testing a different param. We've tried to only have one expect per test if we can. I think we also need to test the default works when the param is not provided

Reply:
Done.